src/lib/mina_base/test/account/dune

```diff
@@ -0,0 +1,31 @@
+(library
+ (name account_test)
```

Comment:
```
File "src/lib/mina_base/test/account/dune", line 2, characters 7-29:
2 |  (name mina_base.account_test)
           ^^^^^^^^^^^^^^^^^^^^^^
Error: "mina_base.account_test" is an invalid library name.
Library names must be non-empty and composed only of the following
characters: 'A'..'Z', 'a'..'z', '_' or '0'..'9'.
Hint: mina_base_account_test would be a correct library name
```
Would `mina_base_account_test` be okay? Sounds a lot less appealing to me, but I guess it's better than nothing…

Reply:
Apologies for the confusion. It's a bit odd in `dune` because there is a difference between the local name of a library, and the public name. The local name can't have `.`, but the public one can. Here is an example of a library that uses this namespacing for the public name while setting a local name that just has an underscore in its place. It reads odd, but this does have the desired effect of fixing the global namespacing problem. https://github.com/MinaProtocol/mina/blob/develop/src/lib/mina_metrics/no_metrics/dune